docs/18_github_actions_ci_integration.md

```diff
@@ -0,0 +1,229 @@
+
+
+## Destination configuration
+
+Since we want to export data to the Metrics Web Application, we should specify the destination `firestore` configuration. Let's consider an example of the `firestore` configuration: 
+
+```yaml
+destination:
+  firestore:
+    firebase_project_id: project_id
+    firebase_user_email: ${{ secrets.FIREBASE_USER_EMAIL }}
+    firebase_user_pass: ${{ secrets.FIREBASE_USER_PASSWORD }}
+    firebase_public_api_key: ${{ secrets.FIREBASE_PUBLIC_API_KEY }}
+    metrics_project_id: awesome_project
+```
+
+So, the `firestore` configuration should contain the following keys: 
+
+| Property Name            | Type   | Description  |
+| ------------------------ |:------:| -------------|
+| firebase_project_id      | Public | A firebase project identifier where to export the data. |
+| firebase_user_email      | Secret | A firebase user email used to log in to the Metrics Web Application. |
+| firebase_user_pass       | Secret | A firebase user password used to log in to the Metrics Web Application. |
+| firebase_public_api_key  | Secret | A public API key that could be created using the Google Cloud Platform in the [API & Services credentials](https://console.cloud.google.com/apis/credentials?project=metrics-d9c67) section. This key should have access to the `Identity Toolkit API`. For more information about Firebase API Keys check [this article](https://firebase.google.com/docs/projects/api-keys). |
+| metrics_project_id       | Public | A firestore document identifier of the project to import data to. |
+
+__*Please, NOTE*__ that the `Secret` values must be stored as [GitHub Secrets](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets). The configuration file can contain this fields as a environment variables. To replace them in the GitHub actions workflow you can use `gettext`  or replace it using the `sed` tool. The example of using the `sed` command provided in the [Metrics Integration Actions](#Metrics-Integration-Actions) section.
+
+Once we've finished creating the configuration file, we should configure the GitHub Actions that will export the build data to the Metrics Web application. 
+
+To export the data, we should configure the following actions: 
+
+- [`Notify about the building project`](#Notify-about-the-building-project) - the job needed to notify the `Metrics Integration Actions` that some project build was started.
+- [`Metrics Integration Actions`](#Metrics-Integration-Actions) - the workflow needed to export the data to the Metrics Web application using the CI Integrations component.
+
+Let's consider each action in more detail.
+
+## Notify about the building project
```

Comment:
What if we put the sequence diagram somewhere around here - so that right after we can explain its steps? Users will be able to see the complete picture first and then explore the steps of interest?